src/System.Runtime.Extensions/src/System/Environment.cs

```diff
@@ -61,41 +61,7 @@ public static void SetEnvironmentVariable(string variable, string value, Environ
             EnvironmentAugments.SetEnvironmentVariable(variable, value, target);
         }
 
-        public static string CommandLine
```

Comment:
I don't see any tests for CommandLine. Are there any? How do we know this isn't breaking or impacting compat?